app/src/main/java/com/globant/scriptsapadea/ui/activities/ScreenPlayActivity.java

```diff
@@ -4,21 +4,20 @@
 import android.content.Context;
 import android.content.Intent;
 import android.os.Bundle;
+import android.support.v4.app.Fragment;
 import android.view.View;
 
 import com.globant.scriptsapadea.R;
-import com.globant.scriptsapadea.ui.fragments.PictureFragment;
+import com.globant.scriptsapadea.interfaces.OnNavigateToFragmentListener;
 import com.globant.scriptsapadea.ui.fragments.ScreenPlayFragment;
-import com.globant.scriptsapadea.ui.fragments.TakePictureFragment;
 
 import roboguice.inject.ContentView;
 
 @ContentView(R.layout.activity_screen)
-public class ScreenPlayActivity extends BaseActivity implements ScreenPlayFragment.OnScreenplayChangeFragmentListener,PictureFragment.OnSetPictureFragmentImageListener  {
+public class ScreenPlayActivity extends BaseActivity implements OnNavigateToFragmentListener {
```

Comment:
@leomendezg it makes no point to have a "generic" listener to reused. Every Listener should have they corresponding Class and this one belongs no one.